### PR TITLE
Fix issue with textColor if a background color is defined

### PR DESCRIPTION
**Pull request by felginep**
This commit fixies #52

### CODE REVIEW

File: Sources/Models/Views/Label.swift [hunk 1]
```diff
@@ -88,7 +91,7 @@ public struct Label: IBDecodable, ViewProtocol {
             subviews:                                  container.childrenIfPresent(of: .subviews),
             text:                                      container.attributeIfPresent(of: .text),
             textAlignment:                             container.attributeIfPresent(of: .textAlignment),
-            textColor:                                 container.elementIfPresent(of: .textColor),
+            textColor:                                 colorsContainer?.withAttributeElement(.key, CodingKeys.textColor.stringValue),
```
Comment (kateinoigakukun) [inline Sources/Models/Views/Label.swift]: I would suggest below to extract `textColor`.
```swift
let colors: [Color] = try container.elements(of: .color)
colors.first { $0.key == "textColor" }
colors.first { $0.key == "backgroundColor" }
```

Comment (felginep) [reply Sources/Models/Views/Label.swift]: I used the same technique as in https://github.com/IBDecodable/IBDecodable/blob/5e07f7b761aed10e8035c34bdae30c44cf5a9755/Sources/Models/Documents/Placeholder.swift.
This was suggested in the related issue #52 .

Comment (kateinoigakukun) [reply Sources/Models/Views/Label.swift]: I see. That makes sense.

**Comment by felginep** [comment]
Do you still need some changes ?

**Comment by kateinoigakukun** [comment]
No need! I just forgot to merge 🙇

**Pull request merged**